src/ray/gcs/actor_state_accessor_test.cc

```diff
@@ -43,7 +43,7 @@ class ActorStateAccessorTest : public ::testing::Test {
   void GenTestData() { GenActorData(); }
 
   void GenActorData() {
-    for (size_t i = 0; i < 2; ++i) {
+    for (size_t i = 0; i < 100; ++i) {
```

Comment:
The test data will be written to redis. The larger the amount of data, the longer the concurrent read and write of the redis buffer, so that it can be verified to a certain extent whether it is normal in the case of multithreading.